### PR TITLE
Templating changes...

### PR DESCRIPTION
**Pull request by JPOak**
#1094 #1091 #1057

@olefredrik @colin-marshall I realized when I went through the above issues that you needed to do them all at once. Also, they affected a lot of little things across many files. Hopefully, this PR will be helpful to get the ball rolling on changing some of the templating discussed (with some review and discussion.)

- The wrapper should be the same across the theme with the main element.
- The innards should mainly be controlled by content and content-page .

### CODE REVIEW

File: template-parts/content.php [hunk 1]
```diff
@@ -10,16 +10,25 @@
 
 ?>
 
-<div id="post-<?php the_ID(); ?>" <?php post_class('blogpost-entry'); ?>>
+<article id="post-<?php the_ID(); ?>" <?php post_class(); ?>>
 	<header>
-		<h2><a href="<?php the_permalink(); ?>"><?php the_title(); ?></a></h2>
+		<h1 class="entry-title"><?php the_title(); ?></h1>
```
Comment (colin-marshall) [inline template-parts/content.php]: This is going to result in multiple h1 elements on archive pages, which we don't want. There needs to be conditionals throughout this file that use `is_single()` to dictate different content depending on if it's a single post or an archive page.

See here for an example:
https://github.com/WordPress/twentyseventeen/blob/master/components/post/content.php

File: archive.php [hunk 2]
```diff
@@ -18,7 +18,7 @@
 get_header(); ?>
 
 <div class="main-wrap" role="main">
```
Comment (colin-marshall) [inline archive.php]: `role="main"` can be dropped from here. It belongs on the main element but we don't need it there since it's applied by default.

Comment (colin-marshall) [inline archive.php]: `role="main"` can be dropped from here. It belongs on the main element but we don't need it there since it's applied by default.

**Comment by olefredrik** [comment]
Tested ok 👍

**Pull request merged**